### PR TITLE
Update delete command

### PR DESCRIPTION
**Pull request by ritvi12**
Update the delete command to delete on the basis of both index and name

### CODE REVIEW

**Comment by OttoTan** [comment]
Other than maybe including some more tests, LGTM

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public void execute_invalidCommandFormat_throwsParseException() {
     @Test
     public void execute_commandExecutionError_throwsCommandException() {
         String deleteCommand = "delete Alex Yeoh";
-        assertCommandException(deleteCommand, MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        assertCommandException(deleteCommand, MESSAGE_INVALID_PERSON_DISPLAYED_NAME);
     }
```
[the remark this answers is not in the document]
Comment (ritvi12) [reply src/test/java/seedu/address/logic/LogicManagerTest.java:66]: have included it in delete command test

File: src/test/java/seedu/address/logic/commands/DeleteCommandTest.java [hunk 2]
```diff
@@ -40,11 +41,11 @@ public void execute_validNameUnfilteredList_success() {
     }
```
[the remark this answers is not in the document]
Comment (ritvi12) [reply src/test/java/seedu/address/logic/commands/DeleteCommandTest.java:43]: need to cause for delete also it uses filter function to find the person to be deleted

**Review by myrilla12 — APPROVED** [review]
Just reviewed most recent changes. LGTM, will merge

**Pull request merged**